protos/sources/vega/api/v1/core.proto

```diff
@@ -405,6 +407,14 @@ message PoWBlockState {
   optional uint64 expected_difficulty = 4;
   // The hashing function used to calculate the block hash
   string hash_function = 5;
+  // The base difficulty for this block for when transactions seen < tx_per_block
+  uint64 difficulty = 6;
```

Comment:
why do we need these 3 params here? i.e. why is expected_difficulty insufficient?

Reply:
so if I'm not mistaken you're not even using expected_difficulty in which case maybe it's not worth bothering with it in the first place.

Reply:
I know, and I asked for it to be added, so thats all on me.